modules/lib/lib-project/src/main/resources/lib/xp/project.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Functions to manage Content Projects.
+ *
+ * @example
+ * var projectLib = require('/lib/xp/project');
+ *
+ * @module project
+ */
+
+/* global __ */
+
+function required(params, name) {
+    var value = params[name];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5556629)